setup.cfg

```diff
@@ -0,0 +1,4 @@
+[pep8]
+max-line-length=999
+ignore=E121,E123,E126,E226,E24,E704,E221,E127,E128,W503,E731,E131,E711,E712
+exclude=test_*
```

Comment:
We shouldn't exclude our test files from PEP8.  We were doing that originally in the core repo because I hadn't yet fixed the unit tests, but now that they're working again we should apply the same coding standards to them as to the rest of the project.  This exclude line was already removed from the core repo, but I'm guessing it was still somewhere else that caused it to get copy/pasted here as well.